src/openrct2/ride/Vehicle.h

```diff
@@ -143,6 +143,7 @@ struct Vehicle : EntityBase
             int8_t ferris_wheel_var_0;
             int8_t ferris_wheel_var_1;
         };
+        int16_t rider_speed_preference;
```

Comment:
Do not add to the union make this a new variable. All variables are to be upper CamelCase

Reply:
Won't this cause issues with saving? Would I then have to change the save format to storethe new value?

Reply:
Having to add an extra field to serialisation isn't much of an issue but yeah you could get away without the property if you tie it to sprite index